shared/headers/create-address.yml

```diff
@@ -0,0 +1,7 @@
+send_date:
+  description:
```

Comment:
W/re [YAML block scalars](https://yaml.org/spec/1.2/spec.html#id2793652), I have found that being explicit saves so much later headache. Unless our source material indicates otherwise, all our block scalars are folded `>`, and we should assume that we are stripping any trailing lines, as well. `-`. (And yes, I have some cleanup to do from earlier work w/re stripping because I was working from Luke's scrape rather than our hbs templates.)